cf-serverd/server_common.c

```diff
@@ -430,123 +443,113 @@ void CfGetFile(ServerFileGetState *args)
         {
             TLSSend(ConnectionInfoSSL(conn_info), sendbuffer, args->buf_size);
         }
+
         return;
     }
 
 /* File transfer */
 
-    if ((fd = safe_open(filename, O_RDONLY)) == -1)
+    int div = 3;
```

Comment:
I know this was existing code but seems like "div" should be named something like "frequency_of_source_file_change_checks" ;) Or something a little shorter. ;)